server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/UserRoutes.java

```diff
@@ -112,8 +117,29 @@ public void define(Service service) {
         defineAllowedFromHeaders();
 
         defineUserExist();
+
+        defineVerifyUsers();
     }
 
+    @POST
+    @Path("/{username}")
+    @ApiOperation(value = "Verifying User Password")
+    @ApiImplicitParams({
+            @ApiImplicitParam(required = true, dataType = "string", name = "username", paramType = "path"),
+            @ApiImplicitParam(required = true, dataType = "org.apache.james.webadmin.dto.VerifyUserRequest", paramType = "body")
+    })
+    @ApiResponses(value = {
+            @ApiResponse(code = HttpStatus.NO_CONTENT_204, message = "Verification successful."),
+            @ApiResponse(code = HttpStatus.UNAUTHORIZED_401, message = "Verification failed."),
+            @ApiResponse(code = HttpStatus.BAD_REQUEST_400, message = "Invalid input user."),
+            @ApiResponse(code = HttpStatus.INTERNAL_SERVER_ERROR_500,
+                    message = "Internal server error - Something went bad on the server side.")
+    })
```

Comment:
On master we got rid of the Swagger documentation that nobody was maintaining.

You may wish to rebase.

Reply:
I noticed that as well, currently working on updating this PR so it works with the current master.

Reply:
Removed the Swagger documentation in the force push below.